.pylintrc

```diff
@@ -70,7 +70,14 @@ disable=missing-docstring,
       wrong-import-order, # Leave this up to isort
       bad-continuation, # Leave this up to black
       line-too-long, # Leave this up to black
-      exec-used
+      exec-used,
+      super-with-arguments, # temp-pylint-upgrade
+      isinstance-second-argument-not-valid-type, # temp-pylint-upgrade
+      raise-missing-from, # temp-pylint-upgrade 
+      unused-argument, # temp-pylint-upgrade  
+      protected-access, # temp-pylint-upgrade 
+      super-init-not-called, # temp-pylint-upgrade
+      invalid-overridden-method, # temp-pylint-upgrade 
```

Comment:
what is is `temp-pylint-upgrade`?

Reply:
Just a comment to note down the rules we disabled when we upgraded pylint. Intention is to come back and fix these lint issues and remove the ignore rules.

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/issues/251